### PR TITLE
Feature/alp 143/onboarding

### PR DESCRIPTION
**Pull request by PAKovalev**

### CODE REVIEW

File: packages/app/src/scenes/auth/pages/SignInAccountPage/components/organisms/ChoiceYourWallet/ChoiceYourWallet.styled.ts [hunk 1]
```diff
@@ -19,9 +26,15 @@ export const Image = styled.img`
 
 export const Link = styled.a`
   font-size: var(--font-size-m);
-  color: #ffffff;
+  color: var(--white);
```
Comment (dmitry-yudakov) [inline packages/app/src/scenes/auth/pages/SignInAccountPage/components/organisms/ChoiceYourWallet/ChoiceYourWallet.styled.ts]: we should probably take it from the theme, not global var.

File: packages/app/src/scenes/AppAuth.tsx [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
-      history.push(ROUTES.signIn, {from: history.location.pathname});
-    }
-  }, [authStore.hasToken, history, sessionStore]);
+  if (pathname === ROUTES.signIn && !sessionStore.isGuest) {
+    history.push(ROUTES.explore);
+  }
+
+  if (pathname === ROUTES.explore && sessionStore.isGuest) {
+    history.push(ROUTES.signIn);
+  }
```
Comment (dmitry-yudakov) [inline packages/app/src/scenes/AppAuth.tsx:20]: We no longer redirect to signIn page if not authenticated according to sessionStore. Not sure if that's ok in all cases.

File: packages/app/src/shared/hooks/useApiHandlers.tsx [hunk 3]
```diff
@@ -44,13 +46,13 @@ export const useApiHandlers = () => {
           break;
 
         case httpErrorCodes.UNAUTHORIZED:
-          document.location = ROUTES.signIn;
+          sessionStore.signOutRedirect();
```
Comment (dmitry-yudakov) [inline packages/app/src/shared/hooks/useApiHandlers.tsx:49]: should we maybe put some `redirect-after-signin` param in order to not lose the original url? could be in different PR though.

**Review by dmitry-yudakov — APPROVED** [review]

File: packages/app/src/scenes/AppAuth.tsx (hunk 2 above)
Comment (PAKovalev) [reply packages/app/src/scenes/AppAuth.tsx:20]: 1. Guest don't have access to visit `Explore page`. 
2. Normal user should be redirected to `Explore page` from `Sign In` page.

File: packages/app/src/scenes/auth/pages/SignInAccountPage/components/organisms/ChoiceYourWallet/ChoiceYourWallet.styled.ts (hunk 1 above)
Comment (PAKovalev) [reply packages/app/src/scenes/auth/pages/SignInAccountPage/components/organisms/ChoiceYourWallet/ChoiceYourWallet.styled.ts]: Yeah, fixed.

**Review by FIZLIN — COMMENTED** [review]
LGTM
